config/Dockerfile

```
@@ -3,21 +3,31 @@
 FROM rust:1.90-slim-bookworm AS builder
 
 WORKDIR /app
+ARG SYSLOG_BUILD_PROFILE=release
 RUN apt-get update && apt-get install -y pkg-config libssl-dev && rm -rf /var/lib/apt/lists/*
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)